### PR TITLE
Update default config for BDD100K dataset

### PR DESCRIPTION
**Pull request by ananno**
Default value `for step_per_epoch` changed according to bdd100k dataset as `step_per_epoch` is updated with number of training data `69863` in bdd100k dataset.

Default learning rate for 2nd step value is changed `2e-2 -> 2e-3` to avoid it.

## Motivation and Context

In default config `lm_fyolo_bdd100k.py` the parameter `step_per_epoch` calculation was wrong for bdd100k dataset.

Default learning rate conventionally use `2e-2` in 2nd step which gives `gradient overflow` effect. The value needs to be updated.

## Description

Default config `lm_fyolo_bdd100k.py` for object_detection task with `lm_fyolo` model has `step_per_epoch` calculation based on subset of the data. It needs to update with actual number of training samples in the bdd100k dataset.

The learning rate value for 2nd step modified to `2e-2 -> 2e-3` and subsequent values are made smaller one order of magnitude to avoid `gradient overflow` effect. Also added one more step in learning rate at `step_per_epoch * 150` steps.

## How has this been tested?

As the test code for `BDD100K` dataset PR is still in review. This change has been tested with running training manually.

## Screenshots (if appropriate):
After modifying the learning rate the training does not show `NaN` in loss values.
Screenshot from 2020-01-24 11-47-22

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by ananno** [comment]
@iizukak San:
Can you please review this PR for Ownership Approval.

**Comment by iizukak** [comment]
@ananno thanks. You changed numbers of dataset. Where is the information about this dataset? I want to know it’s correct value or not.

**Comment by ananno** [comment]
> @ananno thanks. You changed numbers of dataset. Where is the information about this dataset? I want to know it’s correct value or not.

@iizukak San:
This information comes from the log when we run training with BDD100K dataset. The training dataset size is 69863. This is used in the config file.
Screenshot from 2020-02-03 11-54-17

**Comment by ananno** [comment]
@iizukak San:
Can you please review one more time.

**Comment by ananno** [comment]
@iizukak San: Can you please review this PR one more time.

File: blueoil/configs/core/object_detection/lm_fyolo_bdd100k.py [hunk 1]
```diff
@@ -84,11 +84,16 @@
 NETWORK.OPTIMIZER_KWARGS = {"momentum": 0.9}
 NETWORK.LEARNING_RATE_FUNC = tf.train.piecewise_constant
 # In the origianl yolov2 Paper, with a starting learning rate of 10−3, dividing it by 10 at 60 and 90 epochs.
-# Train data num per epoch is 16551
-step_per_epoch = int(16551 / BATCH_SIZE)
+# Train data num per epoch is 69863
+step_per_epoch = int(69863 / BATCH_SIZE)
```
Comment (tsawada) [inline blueoil/configs/core/object_detection/lm_fyolo_bdd100k.py]: use integer division `a // b`

Comment (ananno) [reply blueoil/configs/core/object_detection/lm_fyolo_bdd100k.py]: Fixed

**Comment by ananno** [comment]
@tsawada San : Please review for Readability one more time.
